<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalogTimeSeries_Widget</class>
 <widget class="QWidget" name="AnalogTimeSeries_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analog Time Series Tools</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QLabel" name="label_main_title">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Analog Time Series Data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Section" name="export_section" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="exportSectionLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="filenameLayout">
        <item>
         <widget class="QLabel" name="label_filename">
          <property name="text">
           <string>Output Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filename_edit">
          <property name="text">
           <string>analog_output.csv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="exportTypeLayout">
        <item>
         <widget class="QLabel" name="label_export_type">
          <property name="text">
           <string>Export Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="export_type_combo">
          <item>
           <property name="text">
            <string>CSV</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="exportTypeSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stacked_saver_options">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="CSVAnalogSaver_Widget" name="csv_analog_saver_widget"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVAnalogSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/AnalogTimeSeries/CSV/CSVAnalogSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Section</class>
   <extends>QWidget</extends>
   <header>Collapsible_Widget/Section.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
